Docker/dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM dresscodeswift/heasoft:v6.28.swift
+
+USER root
+
+RUN \
+    apt-get update \
+    && apt-get -y upgrade \
+    && apt-get -y install \
+        pip \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/*
+
+USER heasoft
+
+RUN \
+    # get wcstools
+    wget tdc-www.harvard.edu/software/wcstools/wcstools-3.9.6.tar.gz \
+    && tar -xf wcstools-3.9.6.tar.gz -C /opt/heasoft \
+    && rm wcstools-3.9.6.tar.gz \
+    && cd /opt/heasoft/wcstools-3.9.6 \
+    && make all
+ENV PATH=/opt/heasoft/wcstools-3.9.6/bin:$PATH
```

Comment:
i'm tempted to make another image with this step compiled, then the dresscode image would then just inherit from that image, and only install the dresscode pipeline code. this would save a bit of time, as we wouldn't have to compile this every time. but it means more docker images to maintain.  see: https://github.com/spacetelescope/DRESSCode/issues/22